gfx.cpp

```diff
@@ -2894,7 +2884,9 @@ class GfxInternal
         default:    // kGfxCpuAccess_None
             {
                 GfxKernel const bound_kernel = bound_kernel_;
-                setProgramBuffer(clear_buffer_program_, "OutputBuffer", buffer);
+                GfxBuffer       buffer_temp  = buffer;
+                buffer_temp.setStride(sizeof(uint32_t));
+                setProgramBuffer(clear_buffer_program_, "OutputBuffer", buffer_temp);
```

Comment:
Input buffer is now by ref so needed to create a copy that can be modified by changing stride (same functionaly as what was happening previously with pass by value)